From a984cff795f68f9a55bc212ab27797b7cff6fd39 Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sat, 5 Feb 2022 19:52:56 -0500
Subject: [PATCH 1/2] Avoid msvc dependencies

Forwarded: not-needed

We are not building on a Microsoft platform, so we can safely ignore
these build-deps and configuration choices.

This is a workaround for https://bugs.debian.org/984627
---
 Cargo.toml |  2 --
 build.rs   | 13 -------------
 2 files changed, 15 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index be03fed..07dc4f0 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,5 +39,3 @@ version = "0.3"
 
 [build-dependencies.tempfile]
 version = "3"
-[target."cfg(target_env = \"msvc\")".build-dependencies.vcpkg]
-version = "0.2.9"
diff --git a/build.rs b/build.rs
index 7a7c524..e0dbe66 100644
--- a/build.rs
+++ b/build.rs
@@ -88,19 +88,6 @@ fn check_cv448(includes: &[PathBuf]) -> bool {
                                     "nettle_ed448_shake256_sign"))
 }
 
-#[cfg(target_env = "msvc")]
-fn try_vcpkg() -> Result<Config> {
-    let lib = vcpkg::Config::new()
-        .emit_includes(true)
-        .find_package("nettle")?;
-
-    Ok(Config {
-        have_cv448: check_cv448(&include_paths),
-        include_paths,
-    })
-}
-
-#[cfg(not(target_env = "msvc"))]
 fn try_vcpkg() -> Result<Config> { Err("not applicable")?; unreachable!() }
 
 fn print_library(lib: &pkg_config::Library, mode: &str) {
-- 
2.34.1

